pages/regional_nordic.html

```diff
@@ -1,111 +0,0 @@
----
-layout: page
-permalink: /regions_nordic/
```

Comment:
Can we keep one of these around as an example if we ever get mature enough to have these?

Reply:
Considering our NZ cohort, yeah, we should.

Reply:
Restored by cd584071.